### PR TITLE
ramips: mt7621: enable lzma-loader for netis WF2881

### PR DESCRIPTION
**Pull request by TheNoFace**
Fixes boot loader LZMA decompression issues (LZMA ERROR 1)

### CODE REVIEW

**Comment by neheb** [comment]
why is there a random merge commit?

**Comment by TheNoFace** [comment]
> why is there a random merge commit?

Sorry, upstream was merged into wrong branch.

**Review by neheb — APPROVED** [review]

File: target/linux/ramips/image/mt7621.mk [hunk 1]
```diff
@@ -1680,14 +1680,15 @@ TARGET_DEVICES += netgear_wndr3700-v5
...
+  $(Device/uimage-lzma-loader)
   BLOCKSIZE := 128k
   PAGESIZE := 2048
   FILESYSTEMS := squashfs
   KERNEL_SIZE := 4096k
   IMAGE_SIZE := 129280k
   UBINIZE_OPTS := -E 5
   UIMAGE_NAME := WF2881_0.0.00
-  KERNEL_INITRAMFS := $(KERNEL_DTB) | netis-tail WF2881 | uImage lzma
+  KERNEL_INITRAMFS := $$(KERNEL) | netis-tail WF2881
```
Comment (hauke) [inline target/linux/ramips/image/mt7621.mk:1691]: Why do you change the `KERNEL_INITRAMFS`? 
Did you test initramfs booting?

Comment (TheNoFace) [reply target/linux/ramips/image/mt7621.mk:1691]: Original commit be3e98ce26da5e737744427e1d59dd1c0508edef was merged by pull request [#2763](https://github.com/openwrt/openwrt/pull/2763)

Tested initramfs booting from stock web shell and sysupgrade on initramfs booting.

**Pull request merged**